html-forms/index.html

```diff
@@ -0,0 +1,40 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>HTML Form</title>
+  </head>
+  <body>
+    <h1>Coding Form!</h1>
+    <form>
+      <p>Email: <br />
```

Comment:
This text should be the `label` element.

Reply:
to resolve this, does this mean i change the <p> element to a <label> element?